packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx

```diff
@@ -303,9 +309,9 @@ const GroupedSeriesConfiguration: FC<GroupedSeriesConfigurationProps> = ({
                             });
                         }}
                     />
-                </FormGroup>
+                </Stack>
             )}
-            <GroupSeriesWrapper>
+            <Box bg="gray.1" p={5} sx={{ borderRadius: 3 }}>
```

Comment:
is there a chance we can use `xxs` (4) for example?

also with borderRadius: yuo can use ```sx={(theme) => ({ borderRadius: theme.radius.sm })}` for example

Reply:
Sure. I'll be a bit more careful with these.